src/renderer/components/map/MapLayerFactory.ts

```diff
@@ -37,7 +37,8 @@ export default class MapLayerFactory {
         'line-cap': 'round',
       },
       paint: {
-        'line-color': '#' + color.toString(16),
+        'line-color':
+          color.indexOf('#') > -1 ? color : '#' + color.toString(16),
```

Comment:
colors returned by delta-chat node should always be returned in hex, so the conversion is not needed anymore.

Also `'#' + color.toString(16)` doesn't work by itself anymore since the color generation in the core changed (new master core, so I would just remove the conversion here and rely on deltachat-node doing the conversion right.